Aptoide/aptoide-client-v8
# [ASV-768] Added the new add store dialog as is in Zeplin;

**What does this PR do?**

This PR aims to add the new design for the add stores dialog; To make this work I had to use different size configs for dialog size for land and portrait. This was done on the onResume method due to dialogs behaving weirdly (couldn't even set the match_parent value on the XML);

Also found and fixed a bug where going to the top stores fragment was crashing the app because of reusing a fragment applying themes and having widget/displayable logic (which are incompatible);

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AddStoreDialog.java
- [ ] dialog_add_store.xml

**How should this be manually tested?**

Go to the stores tab and check the add store dialog in both portrait and land (and, if possible, check it out on a tablet too);

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-801](<https://aptoide.atlassian.net/browse/ASV-801>)

**Code Review Checklist**
